playwright-e2e/utils/string-utils.ts

```diff
@@ -8,8 +8,27 @@ export const makeRandomNum = (min = 1, max = 99): number => {
 };
 
 export const makeEmailAlias = (originalEmail: string): string => {
+  if (originalEmail == null || originalEmail.length === 0 || originalEmail.indexOf('@') === -1) {
+    // Evaluate to true if value is: null or undefined or does not contains @
+    throw Error(`Invalid Parameter: Email "${originalEmail}"`);
+  }
   const splintedEmail = originalEmail.split('@');
   const name = splintedEmail[0];
   const domain = splintedEmail[1];
   return `${name}+${Math.floor(Math.random() * 1000000000)}@${domain}`;
 };
+
+/**
+ * Returns the default value if value is null, empty or undefined.
+ * @param value
+ * @param defaultValue
+ */
+export const getEnv = (value: string | undefined, defaultValue: string): string => {
+  if (value == null && defaultValue == null) {
+    throw Error('Invalid Parameters: Value and defaultValue are both undefined or null.');
+  }
+  if (value == null) {
+    return defaultValue;
+  }
+  return value;
```

Comment:
**_#optional_**

lines 30-33 can be shortened like
  `return value || defaultValue;`

Reply:
`||` operator checks whether a value is falsy. The falsy values in JS are: `undefined`, `null`, `0`, `false`, "", `NaN`. 
But I will change this line to `return value == null ? defaultValue : value;`.

Reply:
You are right. 

However, just to explain my point:
We expect that the `value` here either a string or undefined.
It does not make sense to pass numbers(`0`), boolean(`false`), `NaN` there, and even `null`.
If value is an empty string('') we also return defaultValue.